ci/config/releases/2.8-dev.yaml

```diff
@@ -1,5 +1,6 @@
 koji-target-prefix: pulp-2.8
 rsync-target-dir: 2.8/stage
+rsync-tested-dir: 2.8/dev
```

Comment:
Assuming this approach is approved, I'll put together a docs patch to document this new field.